pytx/pytx/common.py

```diff
@@ -592,6 +617,10 @@ def send(cls_or_self,
         if params is None:
             params = {}
         if type_ == 'GET':
+            if request_dict:
+                return Broker.request_dict('GET',
+                                           url,
```

Comment:
The `send()` method allows you to use it whether the object is instantiated or not. For example, both of these could work:

```python
ThreatDescriptor.send()
ThreatDescriptor().send()
```

In each case the URL is determined in a different way. In the former, the `cls._URL` is used for the URL, in the latter the `self._DETAILS` is used for the URL.